worlds/oot_soh/Enums.py

```diff
@@ -3443,6 +3443,8 @@ class Events(str, Enum):
     CAN_EMPTY_BIG_POES = "Can Empty Big Poes"
     CAN_BORROW_MASKS = "Can Borrow Masks"
     BORROW_SKULL_MASK = "Borrow Skull Mask"
+    BORROW_SPOOKY_MASK = "Borrow Spooky Mask"
+    BORROW_RIGHT_MASK = "Borrow Right Mask"
     COULD_PLAY_BOWLING = "Could Play Bombchu Bowling"
```

Comment:
I've added CAN_BORROW_X_MASK events for all the masks. Please use those and remove these ones.

Reply:
I didn't see `CAN_BORROW_RIGHT_MASKS` needed for Mask of Truth, so I added that one.